### PR TITLE
shortened pr template

### PR DESCRIPTION
**Pull request by peasant98**
## What this PR does / why we need it
shortened pr template

## Related Issues

Fixes #

## Some questions

- [X] I have read the contributing guidelines
- [X] I abide by this repository Code of Conduct

## Special notes for your reviewer

(optional)

## Additional Notes

Do you want to add anything else? We :heart: to hear your opinions!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,25 +2,22 @@
 1. If this is your first time, read our contributor guidelines  https://github.com/hackcu/hackcu/blob/master/.github/CONTRIBUTING.md
 -->
 
-## What this PR does / why we need it
+## PR Description and Purpose
 
 A few sentences describing the overall goals of the pull request's commits.
```
Comment (casassg) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: This can be transformed into a comment

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,25 +2,22 @@
...
 <!--(optional) Fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged)-->
 
 Fixes #
 
-## Some questions
-<!-- You can leave this and check them once the PR has been created. -->
 
-- [ ] I have read the contributing guidelines
-- [ ] I abide by this repository Code of Conduct
+<!-- You can leave this and check them once the PR has been created. -->
```
Comment (casassg) [inline .github/PULL_REQUEST_TEMPLATE.md]: This line goes below Miscellaneous title

**Review by casassg — APPROVED** [review]
small nitpicks, lgtm

**Comment by peasant98** [comment]
@casassg you meant commenting out 'PR Description and Purpose', right?

Edit: Oh, 'What this PR does / why we need it' into comment'. I think I understand now.

Edit: Fixed.

**Pull request merged**